panel/models/tabulator.ts

```diff
@@ -374,38 +386,35 @@ export class DataTabulatorView extends HTMLBoxView {
   }
 
   redraw(): void {
-    if (!this._building) {
-      if (this.tabulator.columnManager.element != null) {
-	this.tabulator.columnManager.redraw(true);
-      }
-      if (this.tabulator.rowManager.renderer != null) {
-	this.tabulator.rowManager.redraw(true);
-	this.setStyles()
-      }
+    if (this._building)
+      return
+    if (this.tabulator.columnManager.element != null) {
+      this.tabulator.columnManager.redraw(true);
+    }
+    if (this.tabulator.rowManager.renderer != null) {
+      this.tabulator.rowManager.redraw(true)
+      this.setStyles()
     }
   }
 
   after_layout(): void {
     super.after_layout()
     if (this.tabulator != null && this._initializing)
       this.redraw()
+    this._initializing = false
   }
 
   render(): void {
     super.render()
     this._initializing = true
     const container = div({style: "display: contents;"})
```

Comment:
```suggestion
    const container = div({style: "display: contents"})
```
Very nit, but for consistency with changes below.

Reply:
The one right below:
`const el = div({class: "pnx-tabulator", style: "width: 100%; height: 100%"})`

Reply:
Ah, I'd probably rather change that one.